### PR TITLE
2021 update 3

### PR DESCRIPTION
**Pull request by jonaslybechjensen**

### CODE REVIEW

File: privacy-policy.md [hunk 1]
```diff
@@ -5,11 +5,11 @@ slug: /information/tos/privacy-policy/
...
 
-Effective date: June 25th, 2020
+This policy is effective from 1 March, 2021.
 
 This Privacy Statement (the "Privacy Statement") is provided by:
 
-Video Communication Services AS ("Whereby"/"we")
+Whereby AS ("Whereby"/"we")
 
 Gate 1 no. 101, 6700 Måløy, Norway
```
Comment (ingridod) [inline privacy-policy.md:14]: no. 107

File: privacy-policy.md [hunk 2]
```diff
@@ -39,39 +39,42 @@ and that the content is not violating other users’ privacy.
 
 ## How we collect, process and store information
 
-We in Whereby are committed to safeguarding the privacy of our
+Whereby is committed to safeguarding the privacy of our
 users. Our business model is to provide a paid service to users who
 need additional features on top of the FREE version, and does not rely
```
Comment (ingridod) [inline privacy-policy.md:44]: Free version

File: privacy-policy.md (hunk 1 above)
Comment (ingridod) [inline privacy-policy.md:14]: no. 107

File: privacy-policy.md (hunk 2 above)
Comment (ingridod) [inline privacy-policy.md:44]: Free version

File: privacy-policy.md [hunk 3]
```diff
@@ -122,44 +124,49 @@ the same purpose and legal ground:
...
-> b, and our legitimate interest of communicating important
-> information about your account to you, cf. GDPR art. 6 (1) item f.
+> Service), cf. GDPR art. 6(1) item b, as well as our legitimate interest
+> of handling your requests cf. GDPR art. 6 (1) item f. Processing the
+> information you submit and our responses are also necessary to help
+> our customer support staff give each other feedback and learn, which
+> constitutes our legitimate interest cf. GDPR art. 6 (1) item f.
+
+* **Public professional information.** We may obtain professional 
+  information about you if you have a linkedIn profile. We use this
```
Comment (ingridod) [inline privacy-policy.md:157]: LinkedIn

File: privacy-policy.md [hunk 4]
```diff
@@ -193,6 +200,45 @@ the services privacy policies: [Google Privacy Policy](http://www.google.com/pol
 > fulfilling such a contract (Terms of Service) cf. GDPR art. 6 (1)
 > item b.
 
+### How and when we reach out to you
+  You control what information we send to you by changing your 
+  [Privacy settings](https://whereby.com/user/privacy). You cannot opt out
+  of receing service and transactional notifications (unless you delete 
+  your account) as they are necessary information for the Services
```
Comment (ingridod) [inline privacy-policy.md:207]: Missing a point . at the end?

File: privacy-policy.md [hunk 5]
```diff
@@ -204,12 +250,12 @@ recording in compliance with regulations after downloading it from
 Whereby.
 
 ### Security
-In the FREE version of the Service, users can use “Small meeting” mode
+In the FREE version of the Services, users can use “Small meeting” mode
```
Comment (ingridod) [inline privacy-policy.md:253]: Free

File: privacy-policy.md [hunk 6]
```diff
@@ -311,6 +357,14 @@ where such retention is necessary for compliance with a legal
 obligation to which we are subject, or in order to protect your vital
 interests or the vital interests of another natural person.
 
+## Children
+
+Whereby is only for people 16 years old and over. By using Whereby, 
+you affirm that you are over 16. If we learn that someone under 16 is using
+Whereby, we’ll terminate their account. For use in educational settings or
+use of our Embedded product in your own service, contact us. We encourage
```
Comment (ingridod) [inline privacy-policy.md:365]: Link to https://whereby.com/information/embedded/ on "contact us"?

**Review by ingridod — CHANGES_REQUESTED** [review]
Some smaller things to be changed before merging

**Comment by jonaslybechjensen** [comment]
Cancelled

**Pull request closed without merging**